app/build.gradle

```diff
@@ -86,4 +93,7 @@ task jacocoTestReport(type: JacocoReport, dependsOn: ['testDebugUnitTest', 'crea
 
 connectedCheck {
     finalizedBy jacocoTestReport
-}
\ No newline at end of file
+}
+
+apply plugin: 'com.android.application'
```

Comment:
Correct me if I am wrong but I think we can add these two lines on the top inside plugins in the following manner:
id 'com.android.application' (which can be removed because it is already present)
id 'com.google.gms.google-services'